cfpb/consumerfinance.gov
# Archiving: final sidebar and banner content

Updates archive banner with final content and adds archive-related sidebar content to archived blog, newsroom, and research & reports pages.

## Additions

- Archive sidebar content for Blog/Newsroom and Research & Reports pages



## Changes

- Archive banner content


## How to test this PR
 
1. Check out branch
2. Mark a page as archived in the Blog, Newsroom, and Research & Reports sections
3. Verify that archive banner and sidebar content is displayed correctly on archived pages


## Screenshots

### Blog archived page
<img width="1347" alt="Screen Shot 2020-11-02 at 1 32 29 PM" src="https://user-images.githubusercontent.com/778171/97922200-f0b4e000-1d10-11eb-8a61-64ea7fe091e6.png">

### Newsroom archived page
<img width="1341" alt="Screen Shot 2020-11-02 at 1 33 03 PM" src="https://user-images.githubusercontent.com/778171/97922207-f4486700-1d10-11eb-8b75-edbfbbc84dc7.png">

### Research & reports archived page
<img width="1362" alt="Screen Shot 2020-11-02 at 1 33 36 PM" src="https://user-images.githubusercontent.com/778171/97922302-17731680-1d11-11eb-816b-3778997a6e5f.png">
